src/mesh/region.hh

```diff
@@ -589,13 +600,7 @@ private:
      * Get sets of names set_name_1 and set_name_2 and sort them.
      * Throws ExcUnknownSet if the set with given name does not exist.
      */
-    void prepare_sets( const string & set_name_1, const string & set_name_2, RegionSet & set_1, RegionSet & set_2);
-
-    /**
-     * Read two operands from input array of strings and check if given names
-     * are existing sets. Return pair of checked set names.
-     */
-    pair<string,string> get_and_check_operands(const Input::Array & operands);
+    void prepare_sets( const string & set_name_1, const string & set_name_2, RegionSet & set_1, RegionSet & set_2) const;
 
     /**
      * Create label of region in format: "region_"+id
```

Comment:
Proposed change of add_region methods:
1. read elements from mesh, create regions (ans region sets) with label derived from ID.
2. read Physical names from the MSH file. Finds region by unique ID and dim, call 
RegionDB::rename_region(region, new_label)
3. process region list from mesh input (further rename regions, creating new etc.)
... calls mainly rename_region
.. special case is creating region from element list